addons/account/report/account_invoice_report.py

```diff
@@ -97,10 +97,11 @@ def _select(self):
                 template.categ_id                                           AS product_categ_id,
                 line.quantity / NULLIF(COALESCE(uom_line.factor, 1) / COALESCE(uom_template.factor, 1), 0.0) * (CASE WHEN move.move_type IN ('in_invoice','out_refund','in_receipt') THEN -1 ELSE 1 END)
                                                                             AS quantity,
-                -line.balance * currency_table.rate                         AS price_subtotal,
+                -line.balance * currency_table.rate * (CASE WHEN move.move_type IN ('in_invoice','out_refund','in_receipt') THEN -1 ELSE 1 END)
+                                                                            AS price_subtotal,
```

Comment:
Ok nevermind

We have a discrepancy because the sign of the quantity changes depending on the move_type, but the balance is not changing.

Reply:
So i should revert this line ?

Reply:
Yup